RedHatInsights/compliance-frontend
# chore(tests): Improve coverage reporting

This changes the `coverage.sh` script to also run the test suite (jest + cypress) to gather coverage and combines into a html report under `./coverage`. The three reports are also uploaded to `codecov` with separate flags, which will allow us to compare the reports also on `Codecov`.

It also cleans up includes/excludes for both coverage reports to be aligned and not include unnecessary files.

Babel and other dependencies needed to be updated to have everything working correctly, for some reason.

**How to test:**

1) Run `./coverage.sh`
2) Open the HTML reports in `coverage`, `coverage-cypress` & `coverage-jest`
3) Verify that the report makes sense and the percentages "add up" in the combined report.


## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
